package.json

```diff
@@ -66,7 +66,7 @@
     "chai-fs": "^1.0.0",
     "chai-subset": "^1.6.0",
     "core-js": "^3.0.0",
-    "eslint": "^5.15.2",
+    "eslint": ">=5.15.2",
```

Comment:
It seems as potential source of problems on 7.0 release. Maybe `^5.12.2 || ^6.0` will be more reliable?

Reply:
Yep :+1: